frankenphp.go

```diff
@@ -501,6 +501,72 @@ func ServeHTTP(responseWriter http.ResponseWriter, request *http.Request) error
 	return nil
 }
 
+//export go_putenv
+func go_putenv(str *C.char, length C.int) C.int {
+	// Create a byte slice from C string with a specified length
+	s := C.GoBytes(unsafe.Pointer(str), length)
+
+	// Convert byte slice to string
+	envString := string(s)
+
+	// Check if '=' is present in the string
+	if idx := strings.IndexByte(envString, '='); idx != -1 {
+		// '=' found, set the environment variable
+		key := envString[:idx]
+		val := envString[idx+1:]
```

Comment:
A string like `FOO=` will panic, you need to check if `idx+1` isn't out of range.